src/openloco/map/tilemgr.cpp

```diff
@@ -188,6 +197,7 @@ namespace openloco::map::tilemgr
     // 0x004610F2
     void map_invalidate_selection_rect()
     {
+        // 1=MAP_SELECT_FLAG_ENABLE - from RCT2 code
```

Comment:
Create a new constexpr of all of the flags so that you can replace these comments with flags.